src/project/FileTreeView.js

```diff
@@ -531,6 +531,11 @@ define(function (require, exports, module) {
                 case "gif":
                     fileType = "image";
                     break;
+                case "ogv":
+                case "mp4":
+                case "webm":
```

Comment:
@humphd do you mean try to use the ``_lookupMimeType`` in ``content.js`` to get the filetype? That method doesn't return the same stuff as our amazing switch statement. We'd have to refactor some stuff to make it work.